.github/workflows/ci.yml

```diff
@@ -18,3 +18,4 @@ jobs:
     with:
       craTarget: "examples/complete"
       craGoalIgnoreFile: "cra-tf-validate-ignore-goals.json"
+      craEnvironmentVariables: "TF_VAR_existing_at_instance_crn=crn:v1:bluemix:public:logdnaat:eu-de:a/abac0df06b644a9cabc6e44f55b3880e:b1ef3365-dfbf-4d8f-8ac8-75f4f84d6f4a::"
```

Comment:
@ocofaigh do we want the at instance added as an org level variable so other repos can use it?

Reply:
I can put it in `PERMANENT_AT_INSTANCE_CRN_GE_DEV` 
Usage would be:
```
craEnvironmentVariables: "TF_VAR_existing_at_instance_crn={{ vars.PERMANENT_AT_INSTANCE_CRN_GE_DEV }}"
```

Reply:
hmm, I already added it to https://github.com/terraform-ibm-modules/common-dev-assets/blob/main/common-go-assets/common-permanent-resources.yaml in which the tests will be updated to pull from (Jordan will add in subsequent PR). Is there any way to pull from there and set as org level variable so we only have to main in once place?

Reply:
Yes but would require dev work. Flow read file and update org variables with REST API on a cron or part of the merge pipeline for the source repo https://docs.github.com/en/rest/actions/variables?apiVersion=2022-11-28#create-an-organization-variable

Reply:
Can't we just have the common pipeline execute some logic to pull from the yaml at runtime, since it comes from common-dev-assets which the consuming repo will have? (lets continue this conversation on slack as I am about the merge this PR)